### PR TITLE
Finding intersection node of two linked lists connected in y shape In cpp

### PR DESCRIPTION
**Pull request by rk05-lenavo**
#1247 ISSUE

### CODE REVIEW

**Comment by rk05-lenavo** [comment]
@raksha009 Mam can You Please review this code. If it seems Okay to you Kindly merge my PR.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
take input from the user. add 2 testcases and time and space complexities.

**Comment by rk05-lenavo** [comment]
@tarun26091999 Sir Please take a look.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
check your code again and link it to the folder.

**Comment by rk05-lenavo** [comment]
@tarun26091999 Sir what's wrong in the code and how to link it in the folder I think I have done that.

File: Code/C++/Intersection_point_of_linked_list.cpp [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+//        \ /
+//         3 
+//         |
+//         5
+//         |
+//         7
+
+// Output 3
+
+// Time Complexity: O(m+n)
```
Comment (krishna-NIT) [inline Code/C++/Intersection_point_of_linked_list.cpp:153]: add what is m and n here

**Comment by krishna-NIT** [comment]
Update the Readme which is present at the location (Algo-Tree/Linked_list/readme.md), add this question this at Last of this Readme and arrange these question in alphabetical Order

**Comment by rk05-lenavo** [comment]
@krishna-NIT Sir I have also modified the Linked_Lists/readme.md file https://github.com/Algo-Phantoms/Algo-Tree/pull/1832#issue-628021306. Now sir if it seems okay to you please merge my pull request.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
link it in the folder

**Comment by krishna-NIT** [comment]
@rk05-lenavo  you haven't pushed your changes here, do it in this or and what is m and n?

**Comment by krishna-NIT** [comment]
don't create 2 pr for single issue

**Comment by rk05-lenavo** [comment]
@krishna-NIT @tarun26091999  Sir is it okay now.

**Comment by krishna-NIT** [comment]
> @krishna-NIT @tarun26091999 Sir is it okay now.

No, 8742881915 Message me, here it will be difficult for communicating faster @rk05-lenavo

**Comment by plazzy99** [comment]
> > @krishna-NIT @tarun26091999 Sir is it okay now.
> 
> No, 8742881915 Message me, here it will be difficult for communicating faster @rk05-lenavo 

@krishna-NIT please dont share your personal contact no.

**Comment by krishna-NIT** [comment]
> > > @krishna-NIT @tarun26091999 Sir is it okay now.
> > 
> > 
> > No, 8742881915 Message me, here it will be difficult for communicating faster @rk05-lenavo
> 
> @krishna-NIT please dont share your personal contact no.

ok but there are lot more issues how to convey them?

**Comment by krishna-NIT** [comment]
> > > @krishna-NIT @tarun26091999 Sir is it okay now.
> > 
> > 
> > No, 8742881915 Message me, here it will be difficult for communicating faster @rk05-lenavo
> 
> @krishna-NIT please dont share your personal contact no.

ok , sorry sir

File: Code/C++/Intersection_point_of_linked_list.cpp [hunk 2]
```diff
@@ -0,0 +1,154 @@
+// C++ program to get intersection point of two linked list
```
Comment (krishna-NIT) [inline Code/C++/Intersection_point_of_linked_list.cpp:1]: Describe the issue and Explain your approach in brief

File: Code/C++/Intersection_point_of_linked_list.cpp [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+		else
+			head2->next=NULL;
+	}
+	
+	
+
+	cout << "The node of intersection is " << getIntesectionNode(head1, head2);
+}
+
+// Time Complexity: O(m+n)
```
Comment (krishna-NIT) [inline Code/C++/Intersection_point_of_linked_list.cpp:127]: add what is m and n?

**Comment by rk05-lenavo** [comment]
@krishna-NIT Sir is it okay now.

**Comment by krishna-NIT** [comment]
@rk05-lenavo Nice, just update the readme present in linklist folder.
just add this issue name and its location at the bottom side of the readme

**Comment by rk05-lenavo** [comment]
@krishna-NIT Sir I'm new at github so can you Please tell me how I can update readme.md file from here.

**Comment by plazzy99** [comment]
> @krishna-NIT Sir I'm new at github so can you Please tell me how I can update readme.md file from here.
> 

Follow the pinned issue it contains a video that will help you

**Comment by rk05-lenavo** [comment]
@krishna-NIT Sir is it Okay now.

**Comment by krishna-NIT** [comment]
yes @rk05-lenavo  readme updated properly, just resolve the conflict

**Comment by rk05-lenavo** [comment]
@krishna-NIT Sir is it okay now.

**Comment by krishna-NIT** [comment]
Yes @rk05-lenavo ,
@rish-singhal LGTM, Please Merge it

**Comment by rk05-lenavo** [comment]
@krishna-NIT Sir but till now there is no increment in the leaderboard pull request.

**Pull request merged**